__tests__/app.test.js

```diff
@@ -90,3 +90,46 @@ describe("GET /api/articles/:article_id", () => {
     });
 })
 
+describe("GET /api/articles", () => {
+    test("return array of articles, with the correct properties AND no property body", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            articles = response.body.articles
+            console.log(articles)
+            expect(articles.length).toBe(13)
+            articles.forEach((article) => {
+                expect(article.hasOwnProperty('author')).toBe(true);
```

Comment:
One thing that I think would make this test more secure is also checking that they type of the property value is what you would expect it to be. It's one thing to have all of the correct keys, but do the value types match up?